### DISCUSSION TITLE
Choosing plot size

### DISCUSSION THREAD

**Discussion by Taresin** [Chia-Network/chia-blockchain#456] [Q&A] [answered]
Are there any differences between k-size plot choice?

From the plot times shown by other people, It seems like plot time scales proportionately with the size of the plot.
Just wanted to see if there are any other differences between choosing a single big plot or multiple little plots.

Is there any advantage to choosing one strategy over another?

**Comment by hoffmang9** [comment, marked as answer]
K=32 is the network minimum and yes, the effort to generate a plot goes up by a factor of a little more than 2 as you go larger. K=32 is likely to survive 7-12 years on the chain before we'd have to move to k=33 - and that will have a lot of lead time. You're safe with k=32 and it's the easiest.

**Reply by hoffmang9** [reply]
And the only thing that matters is the amount of space taken up by the final file. Two k=32's are just a little smaller than 1 k=33. (Note that a typo was edited here.)

**Reply by Taresin** [reply]
Thank you for explaining that.
I imagine that you mean 2 x k=32 is slightly smaller than 1 x k=33.

**Reply by hoffmang9** [reply]
You are exactly right and I'm going to edit that for others.

**Reply by galgool** [reply]
So there is no limit or preferred number of plots on one hard drive (as long as there is space)?

**Reply by hoffmang9** [reply]
There are no practical limits on number of plots per drive.

**Reply by galgool** [reply]
Ok, thank you.

And is there a limit on the number of added plot directories in the GUI?
I’m asking this as I want to add quit a few external hard drives to my computer.

**Reply by hoffmang9** [reply]
The only limit is likely in your operating system

**Reply by klapies** [reply]
Hoffmang can you answer the question? Size and earnings difference? I do not mean time of plotting. What is difference betwnn small size plot and biggest plot. I don't mean the size I Asking about winning etc.
thx

**Reply by hoffmang9** [reply]
Asking yet again doesn't get you a different answer. Two k32s will win almost exactly the same as a single k33.

**Reply by klapies** [reply]
ok

**Reply by Tofandel** [reply]
(You didn't fix the typo, now it's 32 instead of 33 :p)

So technically it's better to plot in k=33 if you don't want to have to replot in 10 years since everything else is the same

**Reply by hoffmang9** [reply]
> (You didn't fix the typo, now it's 32 instead of 33 :p)
> 
> 
> 
> So technically it's better to plot in k=33 if you don't want to have to replot in 10 years since everything else is the same

But in 15 years you may realize you wasted effort and k32s have another 10 years.
